packages/node/src/generators/setup-docker/files/Dockerfile__tmpl__

```diff
@@ -0,0 +1,6 @@
+FROM node:alpine
```

Comment:
You probably want something more elaborate in the lines of

```
ROM docker.io/node:lts-alpine

ENV NODE_ENV production

WORKDIR <%= app %>
RUN npm install -g npm@latest

COPY <%= buildLocation %> ./
RUN npm install --omit=dev

RUN chown -R node:node .
USER node

CMD [ "node", "<%= main %>" ]
```

Reply:
`FROM docker.io/node:lts-alpine` makse sense, I wouldn't do `npm install` in `Dockerfile` since the `dist` should already be present andd bundled.

If users have external dependencies (e.g. not bundled in) then it makes sense to include install step -- I think they can add the install it manually for now.

Reply:
Also for can run `COPY --chown node:node <%= buildLocation %> .`

Like:

```
RUN addgroup --system node && \
          adduser --system -G node node

COPY --chown node:node <%= buildLocation %> .
```

Reply:
>  I wouldn't do npm install in Dockerfile since the dist should already be present and bundled.

In my setup (node app / webpack) the deps are not bundled. 
I was not aware of the possibility that deps could be bundled for a node app.

Reply:
Yeah, it's possible if you use esbuild or webpack... not plain tsc though. I highly recommend it since you benefit from a smaller image size as well. :)

Reply:
I wouldn't though tsc does bundling or tree shaking!